conda_build/skeletons/cran.py

```diff
@@ -1226,28 +1252,31 @@ def skeletonize(in_packages, output_dir=".", output_suffix="", add_maintainer=No
             d['skip_os'] = '# no skip'
 
         need_git = is_github_url
-        if cran_package.get("NeedsCompilation", 'no') == 'yes':
```

Comment:
This scan was run originally only for files that explicitly claimed no compilation. But since I need to scan this package for test files regardless, I removed this if/then. This might have the side effect that some packages that _claim_ no compilation are nevertheless given compilation dependencies because of the presence of C/Fortran files in the package. I suspect this is not an issue, but if the reviewer disagrees what I would suggest is to put a check _after_ the pass through the file to zero out `need_c`, etc. for pure-R packages